crates/glaredb/src/local.rs

```diff
@@ -157,26 +156,14 @@ impl LocalSession {
                         }
                     },
                     _ => {
-                        let mut parts = buffer.splitn(2, ';');
-                        let first = parts.next().unwrap();
-                        scratch.push_str(first);
-
-                        let second = parts.next();
-                        if second.is_some() {
-                            match self.execute(&scratch).await {
-                                Ok(_) => {}
-                                Err(e) => println!("Error: {e}"),
-                            };
-                            scratch.clear();
-                        } else {
-                            scratch.push(' ');
-                        }
+                        match self.execute(&buffer).await {
```

Comment:
all of this is handled in the `Validator` now. Anything that makes it this point is already validated as `Complete`